cms/templates/import.html

```diff
@@ -102,8 +102,8 @@ <h3 class="title">
           <ol class="status-progress list-progress">
             <li class="item-progresspoint item-progresspoint-upload is-complete">
               <span class="deco status-visual">
-                <i class="icon fa fa-cog"></i>
-                <i class="icon fa fa-warning"></i>
+                <span class="icon fa fa-cog" aria-hidden="true"></span>
```

Comment:
Actually, it might be necessary. It looks like this after everything passes:

![screen shot 2016-06-03 at 11 37 06 am](https://cloud.githubusercontent.com/assets/6182097/15784234/a032d2a6-297f-11e6-8401-c42cc733c816.png)

There's no way to tell unless you're seeing the colors. So I'll add something like "Success" and "Warning" as sr-text.